cmds/webboot/webboot_test.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	ui "github.com/gizak/termui/v3"
+	"github.com/u-root/webboot/pkg/menu"
+)
+
+func pressKey(ch chan ui.Event, input []string) {
+	var key ui.Event
+	for _, id := range input {
+		key = ui.Event{
+			Type: ui.KeyboardEvent,
+			ID:   id,
+		}
+		ch <- key
+	}
+}
+
+func TestDownload(t *testing.T) {
+	t.Run("error_link", func(t *testing.T) {
+		expected := fmt.Errorf("%q: linkopen only supports file://, https://, and http:// schemes", "errorlink")
+		if err := download("errorlink", "/tmp/test.iso"); err.Error() != expected.Error() {
+			t.Errorf("Error: error msg are wrong, want %+v but get %+v", expected, err)
+		}
+	})
+
+	t.Run("download_tinycore", func(t *testing.T) {
+		fPath := "/tmp/test_tinycore.iso"
+		url := "http://tinycorelinux.net/10.x/x86_64/release/TinyCorePure64-10.1.iso"
+		if err := download(url, fPath); err != nil {
+			t.Errorf("Error: when downloading, get error %+v", err)
```

Comment:
I think Go will already print the error in such a way that its clear that it's an error, so to make the error messages shorter, you could do something like `t.Errorf("failed to download: %v, err)` - its clear what the error means, and by having the colon its clear that you're printing the actual error received. You could probably do that for most of the error messages in this file